src/shared/scrapers/USA/GA/index.js

```diff
@@ -180,19 +181,36 @@ const scraper = {
     'Worth County'
   ],
   async scraper() {
+    const afterChange = datetime.scrapeDateIsAfter('2020-3-26');
+    let selector = 'table:contains(County):contains(Cases) tbody > tr';
+    if (afterChange) {
+      const pageHTML = (await fetch.page(this.url)).html();
+      [this.url] = pageHTML.match(/https:\/\/(.*)\.cloudfront\.net/);
+      selector = 'table:nth-child(6) tbody tr:not(:first-child,:last-child)';
+    }
     const $ = await fetch.page(this.url);
     let counties = [];
-    const $table = $('table:contains(County):contains(Cases)');
-    const $trs = $table.find('tbody > tr');
+    const $trs = $(selector);
     $trs.each((index, tr) => {
       const $tr = $(tr);
-      const name = $tr.find('td:first-child').text();
+      const name = $tr
+        .find('td:first-child')
+        .text()
+        .replace('Mcduffie', 'McDuffie');
```

Comment:
I usually like to do this by defining a county map
```suggestion
        .text();

        name = this._countyMap[name] || name;
```